twoliter/src/project.rs

```diff
@@ -208,6 +258,8 @@ pub(crate) struct Image {
     pub name: String,
     pub version: Version,
     pub vendor: String,
+    #[serde(skip, default)]
+    pub(crate) is_sdk: bool,
```

Comment:
Hmmm, I rather hate this. @jmt-lab says he's going to go back to a version of the resolve function that doesn't require this.

(if we really did need the struct member...)

```suggestion
    pub(crate) image_type: ImageType,
```

```rust
pub(crate) ImageType {
    Sdk,
    Kit,
}
```